dashboard/src/components/jobs/JobHandler.vue

```diff
@@ -156,7 +185,22 @@ Component that handles basic job/workflows.
               :irradiance_type="jobParameters.irradiance_type"
               :data_objects="filteredDataObjects(step)"
             >
-              <b>Upload Predicted Performance</b>
+              <b>upload predicted performance</b>
```

Comment:
Why the case change?  here and below

Reply:
Thanks for catching that. I don't know what did this. I think it slipped in with some linting and broke some of the components too.